src/ui/Folding/Folding.ts

```diff
@@ -79,30 +81,44 @@ export class Folding extends Component {
      * Specifies the name of the field on which to do the folding.
      *
      * Specifying a value for this option is required for this component to work.
+     *
+     * Default value is `@foldingcollection`.
      */
-    field: ComponentOptions.buildFieldOption({ required: true }),
-
+    field: ComponentOptions.buildFieldOption({ defaultValue: '@foldingcollection' }),
     /**
      * Specifies the field that determines whether a certain result is a child of another top result.
-     *
```

Comment:
Why did you remove this line? I feel it's a better practice to have the note separated by an empty line from the first sentence.